services/bettersocial/fcmToken/sendChatNotification.js

```diff
@@ -4,14 +4,25 @@ const {FcmToken, User} = require('../../../databases/models');
 const UsersFunction = require('../../../databases/functions/users');
 
 const sendChatNotification = async (userTargetId, payload) => {
-  const signedUser = await UsersFunction.findSignedUserId(User, userTargetId);
-  const userTargetToken = await FcmTokenFunction.findAllTokenByUserId(FcmToken, signedUser);
+  const signedUser = await UsersFunction.findSignedUserId(User, userTargetId, {
+    shouldReturnObject: true
+  });
+
+  const userTargetToken = await FcmTokenFunction.findAllTokenByUserId(
+    FcmToken,
+    signedUser?.user_id
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `findAllTokenByUserId` function is now called with `signedUser?.user_id` instead of `signedUser`. This change assumes that `signedUser` will always be an object, which might not be the case if `findSignedUserId` doesn't find a match or encounters an error. Consider adding error handling for when `signedUser` is null or undefined.

```diff
- const userTargetToken = await FcmTokenFunction.findAllTokenByUserId(FcmToken, signedUser);
+ const userTargetToken = await FcmTokenFunction.findAllTokenByUserId(
+   FcmToken,
+   signedUser?.user_id
+ );
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->